### PR TITLE
DPWEBCORE-911: Add readmes and move into folder

### PR DESCRIPTION
**Pull request by benwainwright**
We had the template set up, but didn't really have an explanation of what this repo was or how to use it.

### CODE REVIEW

**Review by abrarkhan1234 — APPROVED** [review]
Wonderful!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# devx-templates
+
+This repository contains templates to help with quickly spinning up the boilerplate associated with new projects. The current contents are:
+
+* cosmos-lambda: An AWS lambda project, written in Typescript, deployed via Cosmos
```
Comment (MattLongCode) [inline README.md:5]:
```suggestion
* cosmos-lambda: An AWS lambda project, written in TypeScript, deployed via Cosmos
```

**Review by MattLongCode — APPROVED** [review]

File: templates/cosmos-lambda/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Initial Setup
+
+1. Clone this repository and copy this folder into a new project
+1. Modify the following files to reflect the correct **stack name** and **AWS account number**
+   * `stackDeployConfig.json`
+   * `src/cdk-entry.js`
+   * `src/infrastructure/lambda-stack.ts`
+1. Run `yarn install`
+1. Authenticate the command line with the test AWS account
```
Comment (abrarkhan1234) [inline templates/cosmos-lambda/README.md:18]: Can we give an example of the actual command to do this, ie the `cli-wormhole {aws-account-number}`?

File: templates/cosmos-lambda/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+1. Click on the created function, then for each environment, click on the "Function" button to register your function with that environment
+1. Run `yarn lambda:publish` to create a new version in Cosmos
+1. Run `yarn lambda:deploy:test` to deploy that version to test and/or `yarn lambda:deploy:live` to deploy that version to live
+
+Your project is now bootstrapped. Going forward; any changes made to the infrastructure will need you to run the `yarn stack:deploy:*` commands against each environment. To deploy new versions of the application you simply need to perform the last two steps again.
+
+## Usage Guide
+
+- `yarn install`: Installs required dependencies
+- `yarn stack:deploy:test` and `yarn stack:deploy:live`: This will synthesize the AWS CloudFormation templates from the CDK and deploy them to each environment. Note: Your command line will to be authenticated against the appropriate AWS account.
```
Comment (abrarkhan1234) [inline templates/cosmos-lambda/README.md:33]: > Note: Your command line will to be authenticated

wee typo stray `to`

**Review by abrarkhan1234 — APPROVED** [review]
Couple of minor suggestions should you chose to accept.

File: templates/cosmos-lambda/README.md (hunk 2 above)
Comment (benwainwright) [reply templates/cosmos-lambda/README.md:18]: @abrarkhan1234 I removed that because people do this in all sorts of different ways.

Comment (benwainwright) [reply templates/cosmos-lambda/README.md]: Ok, I've added a link to the `cli-wormhole` repository.

**Pull request merged**